bin/migrate-goldens.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+import { appendFile, mkdir, readFile, rename, rm } from 'node:fs/promises';
+import { join, normalize, parse, sep } from 'node:path';
+import commandLineArgs from 'command-line-args';
+import { glob } from 'glob';
+import { stdout } from 'node:process';
+
+const { pattern = './test/**' } = commandLineArgs({ name: 'pattern', type: String, defaultOption: true }, { partial: true });
+const oldSuffix = 'screenshots/ci/golden';
+const newSuffix = 'golden/chromium';
```

Comment:
Nit, could reuse from the plugin export:

```suggestion
const newSuffix = `${PATHS.GOLDEN}/chromium`;
```